ibis/pyspark/compiler.py

```diff
@@ -14,6 +14,8 @@
 from ibis import interval
 from ibis.pyspark.operations import PySparkTable
 from ibis.spark.compiler import SparkContext, SparkDialect
+from ibis.spark.datatypes import ibis_array_dtype_to_spark_dtype, \
```

Comment:
since i asked for another change, i'll also ask that the `\` be replaced with `(` and `)`